ios/RNSScreenController.mm

```diff
@@ -195,4 +323,148 @@ - (BOOL)prefersHomeIndicatorAutoHidden
 }
 #endif
 
+#pragma mark - Fabric specific
+#if RN_FABRIC_ENABLED
+
+- (void)setViewToSnapshot:(UIView *)snapshot
+{
+  [self.view removeFromSuperview];
+  self.view = snapshot;
+}
+
+- (void)resetViewToScreen
+{
+  [self.view removeFromSuperview];
+  self.view = _initialView;
+}
+
+#pragma mark - Paper specific
+#else
+
+- (void)notifyFinishTransitioning
```

Comment:
I think this method and `findFirstResponder`, `willMoveToParentViewController`, `traverseForScrollView`, `getIndexOfView`, `getParentChildrenCount` may be ready to be used in Fabric right now. It is for sure necessary to check if the behaviors concerning them are correct there.